cfme/intelligence/reports/menus.py

```diff
@@ -10,6 +11,9 @@
 
 from . import CloudIntelReportsView, ReportsMultiBoxSelect
 
+item_title = version.pick({'5.9': 'My Company (All Groups)',
```

Comment:
**REQUIRED:** picking can't happen at the module level, better to turn this into a fn

Reply:
Also is there a reason this cant be put in a common area?

Reply:
Done.

I have turned it into a function and moved it to __init__.py